### ISSUE TITLE
Can anyone suggest me a steps to run this code!

### ISSUE THREAD

**Issue by Dhanuswty** [beerboaa/Color-Classification-CNN#3]

**Comment by beerboaa** [comment]
Hello,
You can run color_net_training.py direcly, however, you need directories for both train and test set.
According to Keras ImageDataGenerator (https://keras.io/preprocessing/image/).
color_net_training.py will load images from the directories automatically.

**Comment by Dhanuswty** [comment]
What are the requirements to run this code?
I used this packages
tensorflow-1.12.0
Keras-2.2.4  i got import errors.

**Comment by Coderash1998** [comment]
Where is the dataset?
